css-transitions/styles.css

```diff
@@ -0,0 +1,53 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: sans-serif;
+  background-color: rgb(245 245 245);
+}
+
+section {
+  max-width: 992px;
+  margin: 2rem auto;
+  display: flex;
+  justify-content: space-evenly;
+}
+
+.custom-button {
+  padding: 0.5rem 1.5rem;
+  border: none;
+  border-radius: 0.25rem;
+  font-size: 1rem;
+  color: white;
+  background-color: blue;
+  transition: background-color 0.5s;
+}
+
+.custom-button:hover {
+  background-color: red;
+}
+
+.card-wrapper {
+  text-align: center;
+}
+
+.card {
+  max-width: 18rem;
+  padding: 0.5rem;
+  margin-bottom: 2rem;
+  border: 1px solid rgb(228 228 228);
+  border-radius: 0.25rem;
+  background-color: white;
+  cursor: pointer;
+  transition: scale(1) 0.5s;
```

Comment:
you don't want `scale(1)` here as it is a value for the transform property, just the time. Then you can add `transform: scale(1)` on another line in the .card selector

Reply:
I see. Got it!